web/admin/src/Admin.js

```diff
@@ -109,13 +115,33 @@ export default class Admin extends PureComponent {
     )
   }
 
+  saveCurrentIndex = (currentIndex) => {
+    this.setState({currentIndex})
+  }
+
   questionsForCurrentSession = () => Object.values(this.state.questionsBySession[this.state.sessionId] || {}).sort((a,b) => a.order - b.order)
 
   onSessionChange = e => this.setState({sessionId: e.target.value})
-  onSessionNameChange = e => this.sessionsRef().child(this.state.sessionId).update({name: e.target.value})
+
+  onSessionNameChange = e => {
+    const currentTitle = e.target.value.trim()
+    const dup = Object.values(this.state.sessions).find(i => i.name.toLowerCase() === currentTitle.toLowerCase())
+    if (currentTitle.length && !dup) {
+      this.publicSessionRef().child(this.state.sessionId).update({name: e.target.value, isDisplayable: true})
+      this.sessionsRef().child(this.state.sessionId).update({name: e.target.value, isDisplayable: true})
+    }
+    else {
+      this.publicSessionRef().child(this.state.sessionId).update({name: e.target.value, isDisplayable: false})
+      this.sessionsRef().child(this.state.sessionId).update({name: e.target.value, isDisplayable: false})
+    }
+  }
   onBackgroundUrlChange = e => this.backgroundUrlRef().set(e.target.value)
-  onSecondsChange = e => this.sessionsRef().child(this.state.sessionId).update({secondsPerQuestion: +e.target.value})
-  createSession = () => this.sessionsRef().push({name: 'New Session', secondsPerQuestion: 30}).then(ref => {
+  onSecondsChange = e => {
```

Comment:
IMO this should be a suggestion rather than forcing the event organizer to use max 60 seconds.  Someone will have a use case for longer.  Also, see https://stackoverflow.com/questions/8354975/how-can-i-limit-possible-inputs-in-a-html5-number-element#8355050

Reply:
I agree. This makes the app less flexible.